test/partitioning.jl

```diff
@@ -196,7 +196,20 @@
   end
 
   @testset "SpatialPredicatePartitioner" begin
-    # TODO
+    g = RegularGrid{Float64}(10,10)
+    
+    # check if each partition has only 1 point
+    sp = SpatialPredicatePartitioner((x,y) -> norm(x-y) < 1.0)
+    s = subsets(partition(g, sp))
+    @test length(s) == 100
+
```

Comment:
Also, you had another test where you wanted to check if the points inside the subsets were indeed inside a ball of radius r. Can you please add it back?

Reply:
As discussed, the predicate must be transitive for that test to make sense, it is fine to leave it out.